onnxmltools/convert/sparkml/operator_converters/imputer.py

```diff
@@ -37,12 +39,12 @@ def convert_imputer(scope, operator, container):
                            name=scope.get_unique_operator_name('Split'),
                            op_version=2,
                            axis=1,
-                           split=range(1, len(operator.output_full_names)))
+                           split=[1] * len(operator.output_full_names))
```

Comment:
I assume this piece of code was never tested.

Reply:
Right. With the original code, the unit test fails with this error:
> [E:onnxruntime:, sequential_executor.cc:339 onnxruntime::SequentialExecutor::Execute] Non-zero status code returned while running Split node. Name:'Split' Status Message: Cannot split using values in 'split' attribute. Axis=1 Input shape={5,2} NumOutputs=2 Num entries in 'split' (must equal number of outputs) was 1 Sum of sizes in 'split' (must equal size of selected axis) was 1
And the Split node looks like this:
<img width="718" alt="image" src="https://user-images.githubusercontent.com/6992432/221006482-b0c460ca-15d1-4c96-aabd-21f66019f625.png">
Notice that the split attribute is set to `[1]`.
Based on the [documentation](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Split-2), the "split" attribute should be set to list of ints for length of each output, so it should be [1, 1] in this case:
<img width="628" alt="image" src="https://user-images.githubusercontent.com/6992432/221007498-928488ed-cf23-42af-b086-f9f28e5efdc9.png">